site/src/components/BreakingChangesComments.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import ReactMarkdown from 'react-markdown';
+import { format, parseJSON } from 'date-fns';
+
+const DateText = ({ label, date }) => (
+  <p style={{ fontStyle: 'italic' }}>
+    {label}: <time dateTime={date}>{format(parseJSON(date), 'd.M.yyyy')}</time>
+  </p>
+);
+
+const BreakingChangesComments = () => {
+  const [comments, setComments] = useState([]);
+
+  useEffect(() => {
+    fetch('https://api.github.com/repos/City-of-Helsinki/helsinki-design-system/issues/495/comments')
```

Comment:
Is the goal to add upcoming breaking changes to that github issue, and then fetch them from there to our doc site? I fear this adds one more manual step to release process; after release (or before release), one must check the github issue and update it. Though we probably won't have breaking changes that often, so probably not going to be a big burden

Reply:
Also Im not sure if it's a good idea to promise upcoming breaking changes. What if we end up not making those breaking changes after more close inspection? Hmm..

Reply:
Good discussion and relevant pondering on the topic :) Yes the goal is that to have one place to update the information about upcoming breaking changes. This brings the same information available on the docsite e.g. for POs and designers who are not that familiar with Github, which is more of a developer tool.

We have been requested to communicate breaking changes beforehand so that the projects will have the possibility to prepare mentally for those. 

Also, communications always require some manual work, this makes it happen in one place only. 

I wouldn't be too worried about not making the breaking changes in the end. It is entirely up to us to decide what and when we say something is going to have breaking changes. The current notes include navigation (which has an epic of 42 fixes waiting to be done) and typography update (which will affect every single component) so I would be surprised if nothing breaks and that would be a very positive surprise.